react-es6-popup/webpack.config.js

```diff
@@ -12,25 +12,22 @@ module.exports = {
   output: {
     // This copies each source entry into the extension dist folder named
     // after its entry config key.
-    path: 'extension/dist',
+    path: path.resolve(__dirname) + '/extension/dist',
```

Comment:
I know the old code was broken on Windows but we should try to make this work on Windows. I think this will do it:

```js
path: path.join(path.resolve(__dirname), 'extension', 'dist'),
```